solution_architecture.md

```diff
@@ -269,9 +269,24 @@ If the back end calls from the mobile applications cannot be spread as evenly as
 
 ## Cross-Border Interoperability
 
-A definite prerequisite for compatibility is that the identifiers of the mobile devices can be matched, i.e. the framework by Apple and Google is being used.
+A definite prerequisite for compatibility is that the identifiers of the mobile devices can be matched, i.e. the GAEN framework by Apple and Google is being used.
 
-Further details will be added as soon as they are available.
+Most European countries are developing similar contact tracing apps, these app may use the common frameworks by Google and Apple, enabling transmission and detection of GAEN format diagnosis keys between devices running different contact tracing applications. 
+Each country has its own separate database of users, in order to coordinate exposure information between countries a common service is required.
+The European Federation Gateway Service (EFGS) handles sharing diagnosis keys between different country's backend servers.
```

Comment:
Suggested rewording:
The European Federation Gateway Service (EFGS) enables interoperability of diagnosis keys between the connected country backend servers.